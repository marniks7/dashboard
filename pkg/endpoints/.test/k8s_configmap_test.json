{
    "apiVersion": "v1",
    "data": {
        "version": "v0.39.0"
    },
    "kind": "ConfigMap",
    "metadata": {
        "annotations": {
            "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"v1\",\"data\":{\"version\":\"v0.39.0\"},\"kind\":\"ConfigMap\",\"metadata\":{\"annotations\":{},\"labels\":{\"app.kubernetes.io/instance\":\"default\",\"app.kubernetes.io/part-of\":\"tekton-pipelines\"},\"name\":\"pipelines-info\",\"namespace\":\"tekton-pipelines\"}}\n"
        },
        "creationTimestamp": "2022-09-04T20:18:20Z",
        "labels": {
            "app.kubernetes.io/instance": "default",
            "app.kubernetes.io/part-of": "tekton-pipelines"
        },
        "name": "pipelines-info",
        "namespace": "tekton-pipelines",
        "resourceVersion": "458",
        "uid": "00554c67-fbe3-4d86-a3d4-3738ab16a300"
    }
}
