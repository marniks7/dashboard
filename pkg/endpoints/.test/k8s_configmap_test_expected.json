{
  "kind": "ConfigMap",
  "apiVersion": "v1",
  "metadata": {
    "name": "pipelines-info",
    "namespace": "tekton-pipelines",
    "labels": {
      "app.kubernetes.io/instance": "default",
      "app.kubernetes.io/part-of": "tekton-pipelines"
    },
    "resourceVersion": "458"
  }
}